### PR TITLE
feature(injector): filename template

### PR DESCRIPTION
**Pull request by kevinchappell**
Added: Customizable filename with placeholder template string

## What it do
Provides additional customization to `manifest.json` filename, allowing developer to remove or customize hash (fingerprint), extension and name.

## Usage:

```javascript
const options = {
  filename: '[name].[hash:8][ext]'
}
// then
new WebpackPwaManifest(options)
```

### filename output
```
manifest.12345678.json
```

### CODE REVIEW

**Comment by arthurbergmz** [comment]
Thank you!

It will be available soon, with a couple of changes that I'll make for the next release.

**Comment by kevinchappell** [comment]
Excellent, thank you!

File: src/injector/index.js [hunk 1]
```diff
@@ -31,11 +31,30 @@ const appleTags = {
   'apple-mobile-web-app-status-bar-style': 'meta'
 }
 
+function createFilename (filenameTemplate, json) {
+  const formatters = [{
+    pattern: /\[hash(:\d{1,2})?\]/gi,
+    value: (match, limit = ':32') => {
+      const hash = generateFingerprint(json)
+      limit = hash.length - parseInt(limit.substr(1), 10)
+      return hash.substr(limit)
```
Comment (kevinchappell) [inline src/injector/index.js]: i made a mistake here. have opened PR to fix it

**Comment by fernandopasik** [comment]
@arthurbergmz would it be possible for a new release to have this fantastic feature?

**Comment by arthurbergmz** [comment]
@fernandopasik Please, update to **v3.5.0**!

**Comment by fernandopasik** [comment]
@arthurbergmz, it worked great! Thanks for the hard work!

**Pull request merged**